packages/teleport/src/lib/tdp/sharedDirectoryManager.ts

```diff
@@ -102,6 +110,8 @@ export class SharedDirectoryManager {
   }
 
   // Writes the bytes in writeData to the file at path starting at offset.
+  // @throws Will throw an error if a directory has not already been initialized via add().
+  // @throws {PathDoesNotExistError} if the pathstr isn't a valid path in the shared directory
```

Comment:
Does vscode pick these up with the `//` style comments? It used to only accept `/** */` style IIRC.